app/assets/javascripts/govuk_publishing_components/accessibility-test.js

```diff
@@ -62,6 +97,36 @@
     )
   }
 
+  var _findParent = function (element, selector) {
+    while (element.tagName !== 'HTML') {
+      if (element.matches(selector)) {
+        return element
+      }
+      element = element.parentNode
+    }
+  }
+
+  var _renderIncompleteWarnings = function (incompleteWarnings) {
+    incompleteWarnings.forEach(function (warning) {
+      warning.selectors.forEach(function (selectorObj) {
+        var activeEl = document.querySelector(selectorObj.selector)
+        var activeElParent = _findParent(activeEl, '[data-module="test-a11y"]')
+        var warningWrapper = activeElParent.querySelector('.component-guide-preview--warning')
```

Comment:
Can we target `[data-module="test-a11y-wrapper"]` so that we're not targeting a presentation class